dunai/stack.yaml

```diff
@@ -1 +1,3 @@
 resolver: lts-11.7
+
+extra-deps: [simple-affine-space-0.1]
```

Comment:
btw, simple-affine-space is on stackage now

Reply:
Yes, then it should be removed from the stack file.